scheduler/filter/README.md

```diff
@@ -79,6 +79,26 @@ Those tags are sourced from `docker info` and currently include:
 * kernelversion
 * operatingsystem
 
+#### Constraint Expression Syntax
+
+Additionally, you can use a not (`!`) to negate and a regular expression in the form of `/regexp/` for specifying a constraint.
+Relative comparisons, `>=` and `<=` are also supported, but they are limited to `string` comparison only.
+
+For example,
+
+* `constraint:name=node1` will match nodes named with `node1`.
```

Comment:
I'm not sure but think that restriction is implemented in docker?  maybe @vieux can answer this.